src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/main_corpus.inc

```diff
@@ -144,99 +212,286 @@
-                nav2_beam_cpu = nav2_max_cpu.loc[case_name].loc[bagfile].loc['beam']['max']
-                nav2_beam_peak_rss = nav2_max_rss.loc[case_name].loc[bagfile].loc['beam']['max']
+                nav2_amcl_likelihood_cpu_util = nav2_amcl_likelihood_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                nav2_amcl_likelihood_peak_rss = nav2_amcl_likelihood_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                nav2_amcl_likelihood_prob_cpu_util = nav2_amcl_likelihood_prob_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                nav2_amcl_likelihood_prob_peak_rss = nav2_amcl_likelihood_prob_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                nav2_amcl_likelihood_beam_skip_cpu_util = nav2_amcl_likelihood_beam_skip_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                nav2_amcl_likelihood_beam_skip_peak_rss = nav2_amcl_likelihood_beam_skip_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                beluga_amcl_likelihood_cpu_util = beluga_amcl_likelihood_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                beluga_amcl_likelihood_peak_rss = beluga_amcl_likelihood_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                beluga_amcl_likelihood_prob_cpu_util = beluga_amcl_likelihood_prob_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                beluga_amcl_likelihood_prob_peak_rss = beluga_amcl_likelihood_prob_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                nav2_amcl_beam_cpu_util = nav2_amcl_beam_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                nav2_amcl_beam_peak_rss = nav2_amcl_beam_peak_rss_data.loc[case_name].loc[bagfile]['max']
+                """ """
+                beluga_amcl_beam_cpu_util = beluga_amcl_beam_cpu_util_data.loc[case_name].loc[bagfile]['max']
+                beluga_amcl_beam_peak_rss = beluga_amcl_beam_peak_rss_data.loc[case_name].loc[bagfile]['max']
             except:
-                beluga_likelihood_cpu = beluga_likelihood_peak_rss = nav2_likelihood_cpu = nav2_likelihood_peak_rss = "???"
-                beluga_beam_cpu = beluga_beam_peak_rss = nav2_beam_cpu = nav2_beam_peak_rss = "???"
+                nav2_amcl_likelihood_cpu_util           = "???"
+                nav2_amcl_likelihood_peak_rss           = "???"
+                nav2_amcl_likelihood_prob_cpu_util      = "???"
+                nav2_amcl_likelihood_prob_peak_rss      = "???"
+                nav2_amcl_likelihood_beam_skip_cpu_util = "???"
+                nav2_amcl_likelihood_beam_skip_peak_rss = "???"
+                beluga_amcl_likelihood_cpu_util         = "???"
+                beluga_amcl_likelihood_peak_rss         = "???"
+                beluga_amcl_likelihood_prob_cpu_util    = "???"
+                beluga_amcl_likelihood_prob_peak_rss    = "???"
```

Comment:
@glpuga meta: we really need a better way to deal with partial benchmark runs than these huge `try...except` blocks.